### ISSUE TITLE
joint training

### ISSUE THREAD

**Issue by misakshoyan** [csbhr/CNLRN#1]
Hi @csbhr.

I have a question.
I see in table 5 that you separately train SRN and NLRN networks and then just use them sequentially, thus reaching PSNR=27.052 db. Then, by joint training, you reach PSNR=27.770 db, which means that at first SRN is trained, then in SRN+NLRN network, SRN is initialized by pre-trained SRN and SRN+NLRN is trained jointly with two learning rates.

What about SRN+NLRN trained from scratch jointly with single learning rate?
I wonder whether you have any comparison results of SRN+NLRN (joint training with pre-trained SRN with two learning rates) and SRN+NLRN(trained from scratch jointly with single learning rate)?
Does your joint training with two learning rates with pre-trained SRN has any advantage over joint training from scratch with single learning rate? 

Thanks in advance,
Misak
